Tasks/Doroshevich/Task2/Task2.py

```diff
@@ -0,0 +1,71 @@
+import datetime
+
+choice = input(
+    "Введите '1'- Для отображения текущего времени\nВведите '2'- Если хотите ввести время с консоли\n")
+
+if not choice.isnumeric() or not 1 <= int(choice) <= 2:
+    print("Некорректно введены данные, попробуйте снова")
+    exit()
+
+if int(choice) == 1:
+    time = f"{datetime.datetime.now().hour}:{datetime.datetime.now().minute}"
+    if len(time.split(':')[1]) == 1:
+        time = time.split(':')[0] + ":0" + time.split(':')[1]
+    elif len(time.split(':')[0]) == 1:
+        time = "0" + time.split(':')[0] + ":" + time.split(':')[1]
+else:
+    time = input("Введите время в формате чч:мм:\n")
+    if len(time) != 5 or time[2] != ':':
+        print("Некорректно введены данные, попробуйте снова")
+        exit()
+
+hour, minute = time.split(':')[0], time.split(':')[1]
+if not hour.isnumeric() or not minute.isnumeric() or not 0 <= int(hour) <= 24 or not 0 <= int(minute) < 60:
```

Comment:
this validation is misplaced, you should validate only user input, not the string that you obtained by formatting values from datetime